### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bajo1207**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,21 +2,33 @@
...
-`python run.py`
+### Installing
+To install the required packages run the following command:
+```
+pip install -r requirements.txt
+```
 
-Generating Documentation:
-`python generate_documentation.py`
+#### Installing on windows
```
Comment (VasLem) [inline README.md:26]: What about Mac?

**Review by VasLem — CHANGES_REQUESTED** [review]
I have also checked again the documentation once again. There is no reference to the notebook that was used to select the parameters of xgboost, in the index page. The notebook is "XGBoost Fitting Using Bayesian Optimization"

**Comment by bajo1207** [comment]
> I have also checked again the documentation once again. There is no reference to the notebook that was used to select the parameters of xgboost, in the index page. The notebook is "XGBoost Fitting Using Bayesian Optimization"

Not on the index page, but in the water_security.classification. Does it need to be on the index page?

File: README.md (hunk 1 above)
Comment (bajo1207) [reply README.md:26]: I'm still figuring out if it works out of the box, the old mac I used was a lost case.. I'm trying it on a newer one now

**Comment by VasLem** [comment]
> > I have also checked again the documentation once again. There is no reference to the notebook that was used to select the parameters of xgboost, in the index page. The notebook is "XGBoost Fitting Using Bayesian Optimization"
> 
> Not on the index page, but in the water_security.classification. Does it need to be on the index page?

Hmm, then maybe mention that the user can access the classification specifics, by mentioning the link to the classification? Because it is not very visible in the front page that there are other notebooks

**Pull request merged**